### PR TITLE
Emergency mode development and other changes

### PR DESCRIPTION
**Pull request by discojapi**
# Changelog
- New emergency mode (WIP)
- Deleted junk .gcode file
- Programmed some of the colors of the LED indicator

### CODE REVIEW

File: esp32/main/main.ino [hunk 1]
```diff
@@ -42,7 +42,7 @@ bool connectToWiFi() {
     Serial.println("Retrying...");
     attempts++;
   }
-  if (WiFi.status() == WL_CONNECTED) {
+  if (WiFi.status() == WL_CONNECTED)
```
Comment (josue-fourier) [inline esp32/main/main.ino]: Missing bracket, syntax error

File: esp32/main/main.ino [hunk 2]
```diff
@@ -29,10 +29,19 @@ const int buzzer = 0;
 const int red = 0;
 const int green = 0;
 const int blue = 0;
-const int rgb[3] = {red, green, blue};
+const int RGB[3] = {red, green, blue};
+
+const int pss = 0; // Primary Secure System pinout
```
Comment (josue-fourier) [inline esp32/main/main.ino]: change variable name. It is not descriptive.

File: esp32/main/main.ino [hunk 3]
```diff
@@ -42,7 +51,7 @@ bool connectToWiFi() {
     Serial.println("Retrying...");
     attempts++;
   }
-  if (WiFi.status() == WL_CONNECTED) 
+  if (WiFi.status() == WL_CONNECTED) {
```
Comment (josue-fourier) [inline esp32/main/main.ino]: Thanks for the bracket back.

File: esp32/main/main.ino [hunk 4]
```diff
@@ -167,12 +180,31 @@ void loop() {
...
+    // Set the status LED color acording the machine status 
+    if (status == 1){
+      rgb_set(0, 1, 0)
+    }
+    else if (status == 2)
+    {
+      rgb_set(0, 0, 1)
+    }
+    if (digitalRead(pss) != HIGH){ // Enters the emergency mode in case the case lid is open
+      status = -1
```
Comment (josue-fourier) [inline esp32/main/main.ino]: missing semicolon, syntax error.

File: esp32/main/main.ino [hunk 5]
```diff
@@ -167,12 +180,31 @@ void loop() {
...
-    String ip = WiFi.localIP().toString();
-    if (sendUUID(uuid, ip)) {
-      delay(200); // Delay to avoid sending data too frequently
+  if (status == -1){
+    rgb_set(1, 0, 0);
+  }
+  else if (status != -1){
+    // Set the status LED color acording the machine status 
+    if (status == 1){
+      rgb_set(0, 1, 0)
```
Comment (josue-fourier) [inline esp32/main/main.ino]: Missing semicolon, syntax error.

File: esp32/main/main.ino [hunk 6]
```diff
@@ -167,12 +180,31 @@ void loop() {
...
+    }
+    else if (status == 2)
+    {
+      rgb_set(0, 0, 1)
```
Comment (josue-fourier) [inline esp32/main/main.ino]: Missing semicolon, syntax error.

File: esp32/main/main.ino [hunk 7]
```diff
@@ -167,12 +180,31 @@ void loop() {
     rgb_set(1, 0, 0);
     connectToWiFi();
   }
-  String uuid = readRFID();
-  if (uuid != "") {
-    String ip = WiFi.localIP().toString();
-    if (sendUUID(uuid, ip)) {
-      delay(200); // Delay to avoid sending data too frequently
+  if (status == -1){
```
Comment (josue-fourier) [inline esp32/main/main.ino]: Add short comments per if/else if.

**Review by n0t-f0ur13r — CHANGES_REQUESTED** [review]

**Review by discojapi — COMMENTED** [review]
TMPFM : Too Much Python For Me.
Correcting

**Review by n0t-f0ur13r — APPROVED** [review]

**Pull request merged**